### ISSUE TITLE
TOOLCHAIN --with-sirius=no adds pugixml library to arch file

### ISSUE THREAD

**Issue by Anyborr** [cp2k/cp2k#3693]
disabling SIRIUS when running the toolchain also disables the downloading and installation of the pugixml library introduced in #3674, but it is still linked in the generated .psmp archfile, which when used to compile causes the error below:

```
...
/usr/bin/ld: cannot find -lpugixml: No such file or directory
collect2: error: ld returned 1 exit status
make[3]: *** [/data/anyber/cp2k/obj/local/psmp/all.dep:146: /data/anyber/cp2k/exe/local/memory_utilities_unittest.psmp] Error 1
make[2]: *** [/data/anyber/cp2k/Makefile:146: all] Error 2
make[1]: *** [/data/anyber/cp2k/Makefile:128: psmp] Error 2
make: *** [Makefile:123: all] Error 2
```

**Comment by stefabat** [comment]
I have the same issue. The reason is that the script file installing pugixml contains some errors, namely, the use of the variable `with_pugixml` which is never defined. Therefore, the CLI options for the library creep in the setup file.

**Comment by mkrack** [comment]
Thanks for reporting the issue. It should be fixed now by PR #3719, please check.

**Comment by Anyborr** [comment]
The compilation with SIRIUS disabled seems to work now. Thanks for fixing the issue!
